Update docs for the CLI and keytool import command (#12544)
MystenLabs/sui: doc/src/learn/cryptography/sui-wallet-specs.md

const address = keypair.getPublicKey().toSuiAddress();
```

> In CLI one can import a mnemonic or a private key from the wallet:

Reviewer: ```suggestion
You can use the CLI to import a mnemonic or a private key from the wallet:
```
